docs/source/working.rst

```diff
@@ -0,0 +1,530 @@
+
+The sections below detail the process of obtaining a linear model with OpenFAST, and will focus on its application to obtain the frequencies and damping of the system modes.
+
+
+
+
+Linearized models for one simulation (manually)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This section describes the key steps to generate a linearized model of the system with OpenFAST. 
+
+The steps to perform simple linearization analysis are given below:
+
+1. Edit the main `.fst` file, set `Linearize=True`
+
+2. Set the output format `OutFmt` to "ES20.11E3". The output files will be written with this high resolution, which is required for accurate eigenvalue analyses.
+
+.. warning::
+    Because the linearization output files are in ASCII format, the results of the eigenvalue analyses will be sensitive to the output resolution (`OutFmt`). It is therefore important to set this parameter with a large precision as mentioned above.
+
+3. There are two main methods to determine at which times the linearization will be made: 
+ 
+   - using `CalcSteady=False`, the user prescribes the times where linearization is to occur using `NLinTimes` and `LinTimes` 
+     (it is the responsibility of the user to provide times where the system is in equilibrium or at a periodic steady state, i.e. sufficiently long time); 
+     - `CalcSteady=True` (recommended approach), OpenFAST will automatically start the linearization when the system is at a periodic steady state (based on given tolerance `TrimTol`) and will perform `NLinTimes` linearizations over a rotor revolution. When a controller is used the option `CalcSteady` will also adjust the controller inputs (either Pitch, Yaw, or Generator Torque, based on the input `TrimCase`) such as to reach the rotational speed indicated by the initial condition. The `TrimGain` and `TrimTol` might need to be adjusted. 
+
+
+
+4. Chose the number of linearizations. For a standstill case, `NLinTimes=1`, for a rotating case, if the equilibrium point is periodic, it is recommended to use `NLinTimes=36`, otherwise `NLinTimes=1`. If `CalcSteady=False` and the user sets `NLinTimes=36`, the user needs to set `LinTimes` with values that corresponds to the rotor being at 36 unique azimuthal position based on the rotor speed. 
+
+
+5. For a typical linearization, the user may set `LinInputs=1`, `LinOutputs=1`, `LinOutJac=False`, `LinOutMod=False`, `Twr_Kdmp=0`, `Bld_Kdmp=0` (see the OpenFAST input file documentation). The standard set of linearization inputs inherent in the linearized system are available when `LinInputs=1`. This includes e.g. collective blade pitch. With `LinOutputs = 1`, every output various `OutList` sections of each module are included in the linearized system. For instance, `GenSpeed` can be included by including `GenSpeed` in the `OutList` of ElastoDyn. Linearization about all the inputs and outputs of OpenFAST set `LinInputs=2`, `LinOutputs=2`, at the expense of having large output files.
```

Comment:
You can also set LinInputs = LinOutputs = 0 to avoid generating the B, C, and D matrices (if the desire is to only use A).